python_modules/dagster/dagster/core/definitions/schedule_definition.py

```diff
@@ -201,6 +201,10 @@ def __init__(
             raise DagsterInvalidDefinitionError(
                 f"Found invalid cron schedule '{self._cron_schedule}' for schedule '{name}''."
             )
+        if len(self._cron_schedule.split(" ")) != 5:
+            raise DagsterInvalidDefinitionError(
+                f"Found non-standard cron schedule '{self._cron_schedule}' for schedule '{name}''."
```

Comment:
nit `non-standard` might be a bit subjective - maybe "Cron schedule {blah} is in an unrecognized format. Dagster cron strings must consist of five characters separated by a single space chacater".